### DISCUSSION TITLE
git: 'start' is not a git command. See 'git --help'.

### DISCUSSION THREAD

**Discussion by iramasauskas** [community/community#34857] [Other Feature Feedback, Questions, & Ideas] [answered]
https://gitexercises.fracz.com/

```
git clone https://gitexercises.fracz.com/git/exercises.git
cd exercises
git config user.name "Ignas"
git config user.email "example@gmail.com"
./configure.sh
git start
```
I copy how to start part, execute it and get this git: 'start' is not a git command. See 'git --help'.
Seems that git start is not working

Any idea how to solve this?
![image](https://user-images.githubusercontent.com/114851822/193449423-40ed78d5-deed-4e70-88cc-49f56c4aed4a.png)

**Comment by airtower-luna** [comment, marked as answer]
> Seems that git start is not working

As the error message says *there is no such command*. This is normal Git behavior when you're trying to call a command that doesn't exist.

From that exercise page it looks like the `./configure.sh` should *create* an alias config for Git that provides a `start` command. But that script is missing on your system, so the command failed (look at the line *above* the one you marked). :warning: 

I can't see why that might be because all the command before that is missing, but the "detached HEAD" warning looks like you did something weird instead of just cloning the repository like the instructions say. :sweat_smile:

**Comment by iramasauskas** [comment]
Yes, the clone did not create` ./configure.sh` in the exercises folder.
I've used now `git checkout master` and this created` ./configure.sh` which allowed to use `git start` now.
Thank you.
